Maquetacion/scss/footer.scss

```diff
@@ -0,0 +1,29 @@
+.footer-bar {
+  align-items: center;
+  background: $blue;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  justify-content: center;
+  margin-top: 30px;
+
+  .footer-text {
+    color: $white;
+    font-size: $text-big;
+    margin: 10px;
+    text-transform: uppercase;
+  }
+
+  .logo-w {
+    margin: 10px;
+  }
+
+  .copy {
+    align-self: flex-end;
+    color: $white;
+    font-size: $text-small;
+    letter-spacing: 1px;
+    padding-right: 20px;
```

Comment:
This is also OK, however I would recommend using padding on container elements. For example, the padding could be applied to the `footer-bar` element instead, restricting all the elements to the `footer-bar` boundaries. Doing this you wouldn't need to apply any kind of padding/margin to this element. 🙂

Reply:
👍